Base/res/js/Spreadsheet/runtime.js

```diff
@@ -197,6 +197,18 @@ class CommonRange {
         });
         return new SplitRange(cellNames);
     }
+
+    unique() {
+        const cellNames = [];
+        const values = new Set();
+        this.forEach(cell => {
+            if (!values.has(cell.value())) {
+                values.add(cell.value());
```

Comment:
This isn't exactly the cheapest operation around, let's do it once
```suggestion
            const value = call.value();
            if (!values.has(value)) {
                values.add(value);
```

Reply:
done :)